foundation-extensions/mobile-core/mobile-core-release-notes.md

```diff
@@ -1,5 +1,15 @@
 # Release Notes
 
+## December 15, 2021
+
+### Android Core 1.9.1
+
+* Fixed an issue which caused the duplicate query parameters in the deep link to be removed. 
```

Comment:
Let's change to:
Fixed an issue that was causing duplicate question parameters in a deep link to be removed.